src/Mizar/Mizar.cpp

```diff
@@ -28,15 +29,24 @@
 
 int main(int argc, char* argv[]) {
   // The main in its current state is used to testing/experimenting and serves no other purpose
-  if (argc < 3) {
-    ORBIT_ERROR("Two filepaths should be given");
+  if (argc < 7) {
+    ORBIT_ERROR(
+        "Example: \n "
+        "$./Mizar path/to/baseline baseline_tid baseline_start_ms path/to/comparison "
+        "comparison_tid comparison_start_ms");
     return 1;
   }
   auto baseline = std::make_unique<orbit_mizar_data::MizarData>();
   auto comparison = std::make_unique<orbit_mizar_data::MizarData>();
 
   const std::filesystem::path baseline_path = argv[1];
-  const std::filesystem::path comparison_path = argv[2];
+  const std::filesystem::path comparison_path = argv[4];
+  const uint32_t baseline_tid = static_cast<uint32_t>(std::stoul(argv[2]));
```

Comment:
We usually use `absl::SimpleAtoi` because it checks the input is valid.

Reply:
I'd like to second this. SimpleAtoi is more consistent and safer in some circumstances.

Reply:
(Alternatively you could also move to Abseil Flags as Davide suggested already. Flags would do all the conversion for you)